test/samples_test.cpp

```diff
@@ -268,3 +267,23 @@ TEST(Samples, SelfIntersect) {
   manifold::PolygonParams().processOverlaps = false;
 }
 #endif
+
+TEST(Samples, CondensedMatter16) {
+  // FIXME: it should be geometrically valid
+  manifold::PolygonParams().processOverlaps = true;
+  Manifold cm = CondensedMatter(16);
+  CheckGL(cm);
+  // FIXME: normals should be correct
+  // CheckNormals(cm);
+  manifold::PolygonParams().processOverlaps = false;
+}
+
+TEST(Samples, CondensedMatter64) {
+  // FIXME: it should be geometrically valid
+  manifold::PolygonParams().processOverlaps = true;
+  Manifold cm = CondensedMatter(64);
+  CheckGL(cm);
+  // FIXME: normals should be correct
+  // CheckNormals(cm);
```

Comment:
Not too surprising some normals are wrong if it's generating overlapped output. Thanks for the note.